src/ext/java/org/opentripplanner/ext/siri/SiriAlertsUpdateHandler.java

```diff
@@ -1,12 +1,15 @@
 package org.opentripplanner.ext.siri;
 
-import org.apache.commons.lang3.tuple.Pair;
 import org.opentripplanner.model.FeedScopedId;
 import org.opentripplanner.model.Route;
 import org.opentripplanner.model.calendar.ServiceDate;
-import org.opentripplanner.routing.alertpatch.*;
+import org.opentripplanner.routing.alertpatch.AlertUrl;
+import org.opentripplanner.routing.alertpatch.EntitySelector;
+import org.opentripplanner.routing.alertpatch.StopCondition;
+import org.opentripplanner.routing.alertpatch.TimePeriod;
+import org.opentripplanner.routing.alertpatch.TransitAlert;
 import org.opentripplanner.routing.core.TraverseMode;
```

Comment:
@lassetyr Would you mind to review this part? The changes to the Siri updaters.

Reply:
Looks good, I am also in favor of merging.